extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -103,7 +130,10 @@ export class AnchorAdStrategy {
     const stickyAd = createElementWithAttributes(
       doc,
       'amp-sticky-ad',
-      dict({'layout': 'nodisplay'})
+      dict({
+        'layout': 'nodisplay',
+        'data-no-fill': String(this.hasOptInStatusNoFillAnchorAds_()),
```

Comment:
I think in the case where OPT_IN_STATUS_ANCHOR_ADS and OPT_IN_STATUS_NO_FILL_ANCHOR_ADS are both present we should insert a fillable anchor ad, since this would be the pubs wish and it also achieves everything that OPT_IN_STATUS_NO_FILL_ANCHOR_ADS achieves in terms of triggering an ad request. (my suggestion above would fix this.)

Reply:
These two OptInStatus should be exclusive but covered that case by suggested logic.